### ISSUE TITLE
Create mercurial repository at Bitbucket

### ISSUE THREAD

**Issue by oheim** [catch22/octave-doctest#97]
I have created a milestone for the integration project. We will probably manage particular tasks as issues here at Github. 

I would really like to work on the integrating topic, so you could concentrate on new features for the Octave Forge package if you wish. @catch22, @cbm755, what are your plans?

**Comment by cbm755** [comment]
I'd be happy for you to work on integration: will help if I can.  @catch22 I assume you're not opposed to a merge to core?

There are some things I don't like about `test()` so a chance to fix those too!  Off the top of my head: it goes into `private/` dirs---not bad per se, but will need messing with PATH or CWD to work I think.

I don't have many long terms plans for this, other than try to fix bugs as we find them.  I was thinking of reformatting the documentation to texinfo before next release.

IIRC @catch22 wanted to refactor some of the code so if anything like that is happening, we maybe should coordinate.

**Comment by cbm755** [comment]
Would you like me to create a Octave hg repo at Bitbucket? (I have academic account)

**Comment by catch22** [comment]
Hi guys -- I'll get back to you & the Octave ML before the end of the day.

**Comment by catch22** [comment]
Can you guys quickly bring me up to speed what is the plan with regards to the hg repository?

**Comment by cbm755** [comment]
1. create clone of *octave* (not doctest)
2. work doctest into that
3. merge back to main octave hg repo

(this way we don't need commit access to octave)

**Comment by catch22** [comment]
Oh, I understand - this is for eventually merging doctest into Octave core. Let's perhaps conclude the discussion on the octave-maintainers ML before taking any action (see also my email there from a couple of minutes ago).

**Comment by oheim** [comment]
You are right, we should fix any rough edges before starting work on the integration task. Also doing the texinfo conversion and refactoring should be done first.

**Comment by cbm755** [comment]
Soon Bitbucket won't be doing mercurial any more, so at least as titled, this issue is out-of-date.

If someone wants to take another look at how we might merge into core Octave, that's of course welcome, but for now I'll close this.  Feel free to re-open in the future.
